packages/components/tabs/src/tabs.js

```diff
@@ -0,0 +1,84 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './tabs.css';
+import { customEventNames } from './utils';
+
+customElementDefineHelper(
+	'ts-tabs',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				// tab properties: {label, icon, selected, closeable, counter}
+				tabs: { type: Array, reflect: true },
+				dir: { type: String, reflect: true }
+			};
+		}
+
+		get direction() {
+			return this.dir ? this.dir : this.bodyDir;
+		}
+
+		tabClickHandler(tab) {
+			console.log('tab', tab);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS2228)